README.md

```diff
@@ -20,98 +17,26 @@ To run this CLI application
-
-Visit your app on: `http://localhost:3000`. You can interact with your smart contract using the `Debug Contracts` page. You can tweak the app config in `packages/nextjs/scaffold.config.ts`.
-
-Run smart contract test with `yarn hardhat:test` or `yarn foundry:test` depending of your solidity framework.
-
-**What's next**:
-
-- Edit your smart contract `YourContract.sol` in `packages/hardhat/contracts`
-- Edit your frontend homepage at `packages/nextjs/app/page.tsx`. For guidance on [routing](https://nextjs.org/docs/app/building-your-application/routing/defining-routes) and configuring [pages/layouts](https://nextjs.org/docs/app/building-your-application/routing/pages-and-layouts) checkout the Next.js documentation.
-- Edit your deployment scripts in `packages/hardhat/deploy`
-- Edit your smart contract test in: `packages/hardhat/test`. To run test use `yarn hardhat:test`
-
-## Documentation
-
-Visit our [docs](https://docs.scaffoldeth.io) to learn how to start building with Scaffold-ETH 2.
-
-To know more about its features, check out our [website](https://scaffoldeth.io).
-
-## Contributing to Scaffold-ETH 2
-
-We welcome contributions to Scaffold-ETH 2!
-
-Please see [CONTRIBUTING.MD](https://github.com/scaffold-eth/scaffold-eth-2/blob/main/CONTRIBUTING.md) for more information and guidelines for contributing to Scaffold-ETH 2.
+## MVP
+- ETT CLI enables users to see the full scope of the tech tree
+- They can see clearly which ones are unlocked and which ones are still locked
+- It only allows them to interact with the challenges that are unlocked
+- It shows them their progress/general score on each challenge and total score on each branch of the tree
+- They can download a challenge locally
+- They can deploy and verify their contract with very few steps
+- They can submit the contract address for the challenge and get real-time feedback
+- If they pass the challenge it unlocks the next set of challenges
+- There are 5 - 10 challenges available. We will limit to a few branches at the start
+- CLI will show leaderboard
+
+## Future Ambitions
+- Issue onchain attestation for completion of the challenge
+- Front end that shows their progress and a leaderboard - phase 1
+- Front end that offers full functionality outside of the ETT CLI - phase 2
+- Add the rest of challenges + add new ones + add new branches
+- Add the other challenge types ("references" which require quizzes, "personal-challenges" which are very big tasks with little instruction - hard for us to test)
+
+## Ideas
+- Add a way for users to prove they have mastered certain concepts so that they can unlock harder challenges earlier 
+- Granular testing so that a user is awarded points based on additional tests (also may need to allow users to go back and resubmit challenges)
+- Capture the flag type NFT that a user can "steal" when they are the new leader. Might consider 1st, 2nd, 3rd place NFTs. Announcing a steal on social media would be great for encouraging user engagement
```

Comment:
Cool gamification, but other than street cred what else is offered with this? Perhaps this is how they get into a cohort, or something.

Reply:
Yes, this will definitely be used for sending builders who are great people but lack the skill to put them in a cohort right away. The hope is that as they develop a portfolio of skills and topic they have mastered which then becomes something that lands them new opportunities inside and outside the Guidl.